src/components/Layout/Header/styles.ts

```diff
@@ -0,0 +1,64 @@
+import { makeStyles } from '@material-ui/styles'
+import { Theme } from '@material-ui/core/styles'
+
+const drawerWidth = 240
+
+export default makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    flexGrow: 1
+  },
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+  appBar: {
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen
+    })
+  },
+  appBarShift: {
+    width: `calc(100% - ${drawerWidth}px)`,
+    marginLeft: drawerWidth,
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen
+    })
+  },
+  hide: {
+    display: 'none'
+  },
+  drawer: {
```

Comment:
Once u create a component folder for the Drawer AKA Menu, move these styles to that folder too!